docs/reference/ml/df-analytics/apis/dfanalyticsresources.asciidoc

```diff
@@ -124,4 +123,67 @@ An {oldetection} configuration object has the following properties:
 `feature_influence_threshold`:: 
   (double) The minimum {olscore} that a document needs to have in order to 
   calculate its {fiscore}. 
-  Value range: 0-1 (`0.1` by default).
\ No newline at end of file
+  Value range: 0-1 (`0.1` by default).
+  
+
+[discrete]
+[[regression-resources]]
+==== {regression-cap} configuration objects
+
+A `{regression}` configuration object has the following properties:
+
+`dependent_variable`::
+  (Required, string) Defines which field of the {dataframe} is to be predicted. 
+  The data type of the field must be numeric. It is also known as continuous 
+  target variable.
+
+`eta`::
+  (Optional, double) The shrinkage applied to the weights. 
+
+`feature_bag_fraction`::
+  (Optional, double) Defines the fraction of features which will be used when 
+  selecting a random bag for each candidate split. 
+
+`gamma`::
```

Comment:
I think it might be worth expanding the documentation regarding the effects of this parameter and a link to the xgboost paper. I think we're also missing a section on our default behaviour, i.e. the hyperparameter optimisation we perform if no parameters are supplied. Maybe that could be referenced from here.